deploy/kube-config/influxdb/influxdb-deployment.yaml

```diff
@@ -16,7 +16,7 @@ spec:
         emptyDir: {}
       containers:
       - name: influxdb
-        image: kubernetes/heapster_influxdb:v0.6
+        image: gcr.io/google_containers/heapster_influxdb:v0.7
```

Comment:
This is totally outdated and will look otherwise when he has rebased, but yes a new image has to be pushed.